third_party/3/Cython/Shadow.pyi

```diff
@@ -0,0 +1,83 @@
+from builtins import (int as py_int, float as py_float,
+                      bool as py_bool, str as py_str)
+from typing import (Union, Dict, Any, Sequence, Optional,
+                    List, TypeVar, Type, Generic)
+
+int = py_int
+long = py_int
+longlong = py_int
+short = py_int
+char = py_int
+sint = py_int
+slong = py_int
+slonglong = py_int
+sshort = py_int
+schar = py_int
+uint = py_int
+ulong = py_int
+ulonglong = py_int
+ushort = py_int
+uchar = py_int
+size_t = py_int
+Py_ssize_t = py_int
+float = py_float
+double = py_float
+longdouble = py_float
+bint = py_bool
+void = Union[None]
+basestring = py_str
+
+gs: Dict[str, Any]  # Should match the return type of globals()
```

Comment:
Ah, legacy... You forget to strip something from a namespace and people will just start picking it up and using it in their code. I didn't write that code but I have hight doubts that it was intended to keep it in there. It does not have a counterpart in Cython's own namespace. Let's see if @robertwb has an opinion on this, but I guess we'll have to leave it in there then...